server.js

```diff
@@ -14,10 +17,60 @@ const welcomeMessage = {
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
 
-app.get("/", function (request, response) {
-  response.sendFile(__dirname + "/index.html");
+let messages = [welcomeMessage];
+
+app.get("/", function (req, res) {
+  res.sendFile(__dirname + "/index.html");
+  // res.json("I am here!");
+});
+
+// Search text in messages -  /messages/search?text=express
+app.get("/messages/search", (req, res) => {
+  const {text }= req.query;
```

Comment:
Well done for doing the search route! Remember to use 'autofix on save' in VS Code to fix things like this spacing - it should be `const { text } = req.query;`